packages/blast-cmd/commands.js

```diff
@@ -111,7 +111,7 @@ const keysInfo = {
           alias: 't',
           type: 'string',
           required: true,
-          describe: 'Amount of tokens in the format 10000000acudos'
+          describe: 'Amount of tokens in acudos'
```

Comment:
can we amke the example describing that we have also a number begore the `acudos` something like `<number>adcudos`